R/cnd-abort.R

```diff
@@ -170,7 +168,7 @@
 abort <- function(message = NULL,
                   class = NULL,
                   ...,
-                  call = NULL,
+                  call = missing_arg(),
```

Comment:
Why the explicit `missing_arg()`?

Reply:
I think I found it weird to not have a default for an argument on the RHS of `...`. I don't have any strong opinion about this though.

Reply:
I think it looks slightly better to me to leave it as `call`, but I don't feel particularly strongly about it.

Reply:
I removed the explicit default.